src/app/adapters/pluginAdapter.js

```diff
@@ -30,6 +31,6 @@ export interface StaticPluginAdapter {
 
 export interface DynamicPluginAdapter {
   graph(): Graph;
-  nodeDescription(NodeAddressT): string;
+  headline(NodeAddressT): ReactNode;
```

Comment:
> ```js
> headline(NodeAddressT): ReactNode;
> ```

I don’t think that this is the right API. It’s more typical to return a
`React.ComponentType<{|+address: NodeAddressT|}>`. This also has the
benefit that clients can provide a stateful implementation if they want.